ui/views.py

```diff
@@ -79,22 +79,22 @@ def stream(request):
     dropbox_files = serializer.validated_data
     response = {}
     for dropbox_file in dropbox_files:
-        video, _ = Video.objects.get_or_create(
-            s3_object_key=dropbox_file["name"],
-            defaults={
-                "source_url": dropbox_file["link"],
-                "creator": request.user,
-            }
+        video = Video.objects.create(
+            source_url=dropbox_file["link"],
+            creator=request.user,
+            title=dropbox_file["name"]
```

Comment:
The upload will continue via celery but no VideoFile object will be created for it (and currently an 'Error' popup shows on the screen.  Maybe it would be better to try creating the VideoFile object before the upload starts, immediately after the Video object is created, and if an exception is raised here for some reason, catch it, delete the Video object, then raise the exception and don't proceed with the upload.